### PR TITLE
Svg

### PR DESCRIPTION
**Pull request by daopunk**
need to finish visual, but it renders

### CODE REVIEW

File: src/contracts/proxies/Vault721.sol [hunk 1]
```diff
@@ -3,25 +3,20 @@ pragma solidity 0.8.19;
...
-  ISAFEEngine public safeEngine;
-  IOracleRelayer public oracleRelayer;
-  ITaxCollector public taxCollector;
-  ICollateralJoinFactory public collateralJoinFactory;
+  IODSafeManager public safeManager;
+  NFTRenderer public nftRenderer;
 
-  string public contractMetaData = "{'name': 'Open Dollar Vaults','description': 'Tradable Vaults for the Open Dollar stablecoin protocol. Caution! Trading this NFT means trading the ownership of your Vault in the Open Dollar protocol and all of the assets/collateral inside each Vault.','image': 'opendollar.com/logo.png','external_link': 'opendollar.com'}";
+  string public contractMetaData =
+    '{"name": "Open Dollar Vaults","description": "Tradable Vaults for the Open Dollar stablecoin protocol. Caution! Trading this NFT means trading the ownership of your Vault in the Open Dollar protocol and all of the assets/collateral inside each Vault.","image": "opendollar.com/logo.png","external_link": "opendollar.com"}';
```
Comment (pi0neerpat) [inline src/contracts/proxies/Vault721.sol:19]: image and externalLink need "https://" protocol

File: parts.txt
Comment (pi0neerpat) [inline parts.txt:1]: Delete this file

**Review by pi0neerpat — CHANGES_REQUESTED** [review]

**Review by pi0neerpat — COMMENTED** [review]
Looks good! Lets deploy

**Review by pi0neerpat — CHANGES_REQUESTED** [review]
1. Variable names in `GoerliContracts.s.sol` have been modified. Is this the new naming schema? I updated my script `prepareForSdk.js` in another branch, and if you think there will be additional changes, let me know.
2. Please delete the old  `GoerliContracts-2.s.sol`. Its confusing to have it here. If you'd like to save these, they should be placed elsewhere and labeled by timestamp.
3. In `GoerliContracts.s.sol`, please add the missing values for these contracts:
- WETH address
- FTRG address
- PROXY_BASIC_ACTIONS
- PROXY_GLOBAL_SETTLEMENT_ACTIONS
4. `yarn build` is failing due to the changes mentioned in item 1 above, specifically in how the contracts are used in `GoerliDeployment.sol`

**Comment by daopunk** [comment]
solves #82

**Review by pi0neerpat — APPROVED** [review]

**Pull request merged**